internal/directives/schemas/git-commit-config.json

```diff
@@ -36,7 +36,6 @@
     "messageFromSteps": {
       "type": "array",
       "description": "TODO",
-      "minItems": 1,
```

Comment:
@hiddeco FYI I removed this as this blocked UI form with 0 items of this attribute.

Reply:
JSON schema is weird sometimes. Min items only applies if the field is _not_ null and _not_ undefined. So the rule basically says, "If you define this field at all, it should be non-empty." idk how realistic this is, but can you send `null` instead of `[]` of the array is empty?

Reply:
Agreed and thats the reason I removed [some properties](https://github.com/akuity/kargo/blob/ca6f53cb60f9448423b47bd1275f6ae789b8bb7f/ui/hack/generate-directive-schema.mjs#L66) that conflicts with what the UI library (rsjf) interprets for the schema. I will put back this attribute as it is and skip validation itself for now and come back later when I have generic solution.